libdnf/error.hpp

```diff
@@ -14,6 +14,7 @@ namespace libdnf {
 class Error : public std::runtime_error {
 public:
     Error(const std::string & what) : runtime_error(what) {}
+    Error(const char * what) : runtime_error(what) {}
```

Comment:
What do you think about replacing the two lines above with one line: 
`using runtime_error::runtime_error;`

Reply:
Right, for some reason I thought inheriting constructors is a C++14 feature? I'll update it.

Reply:
Constructors inheritance is available from C++11. So we can use it here.